deploy_widget.sh

```diff
@@ -4,12 +4,12 @@ then
   deploy_dir=dist_widget
 fi
 echo "Deploying to $deploy_dir"
-if [ ! -e $deploy_dir ]
-then
-  mkdir $deploy_dir
-  mkdir $deploy_dir/dist
-fi
+
+mkdir -p $deploy_dir/dist
+mkdir -p $deploy_dir/../vendor/d3-3.4.13/ # TODO this doesn't seem right but that's the value in app/js/arethusa.dep_tree/services/dep_tree.js
+
 cp dist/arethusa.min.js $deploy_dir
 cp dist/arethusa.min.map $deploy_dir
 cp dist/arethusa_packages.min.js $deploy_dir/arethusa.packages.min.js
 cp dist/* $deploy_dir/dist
+cp vendor/d3-3.4.13/d3.js $deploy_dir/../vendor/d3-3.4.13/d3.js # TODO should this not be d3.min.js?
```

Comment:
Related to the comment above, `dep_tree.js` is requiring `../../vendor/d3-3.4.13/d3.js`. Should that be changed to `d3.min.js`?

Reply:
We probably should use the minified file. As with above, I was just trying to make arethusa match what was in the treebank template.